lucien-ios/LoginScreen/backgroundExtension.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  backgroundExtension.swift
+//  lucien-ios
+//
+//  Created by Fang, Gracie on 9/27/17.
+//  Copyright © 2017 Intrepid Pursuits. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+extension UIView {
+    func addBackground() {
```

Comment:
What's your reasoning for making this an extension?

Reply:
I was trying to fix the repeating patterns by filling the screen with the background image, but I could probably should have put that function in the view controller. Is it better practice to do it that way?

Reply:
Just deleted!